zshipko/ocaml-rs
# ocaml-interop updates

Update to latest ocaml-interop

Closes #52 
Closes #55 

Major changes:
- Remove `frame!` macro completely
- New implementation for `Value` based on a new [Root](https://github.com/zshipko/ocaml-rs/blob/interop-next/src/root.rs) type (similar to `ocaml_interop::BoxRoot` but untyped)
- `Value` no longer implements `Copy`
- Updated macros to convert from `ocaml_sys::Value` to `ocaml::Value` at call boundaries
- Use `ocaml_interop` for runtime initialization
